### ISSUE TITLE
Fix `value-no-vendor-prefix` false positives for `-webkit-box`

### ISSUE THREAD

**Issue by CurrrBell** [stylelint/stylelint#6372]
### What is the problem you're trying to solve?

I recently applied a fairly straightforward stylelint config to a legacy project and found that several instances of `display: -webkit-box;` got changed to `display: box;`, as I would expect with the `value-no-vendor-prefix` rule enabled. 

However, to my surprise not only is there not an equivalent non-prefixed `display` value available, `display: -webkit-box;` and the related `-webkit-line-clamp` property have been [added to the CSS Overflow Module 3 spec](https://drafts.csswg.org/css-overflow-3/#webkit-line-clamp) in that same vendor-prefixed syntax. As far as I can tell, `display: -webkit-box;` is the only valid way (even in non-webkit browsers) to produce the line `-webkit-line-clamp` behavior at the time of writing.

I was able to avoid this autofixing by adding
```
"value-no-vendor-prefix": [
    true,
    {
        "ignoreValues": ["box"]
    }
]
```
to my `.stylelintrc.json`, as the docs correctly suggest.

### What solution would you like to see?

Have `"ignoreValues": ["box"]` be the default value of `value-no-vendor-prefix`

**Comment by ybiquitous** [comment]
@CurrrBell Thanks for using the template.

This issue seems related to not only [`value-no-vendor-prefix`](https://stylelint.io/user-guide/rules/list/value-no-vendor-prefix) but also [`property-no-vendor-prefix`](https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix) because the `-webkit-line-clamp` property has also been standardized for legacy compatibility.

I'm concerned that giving built-in rules some exceptions would lead to less flexibility. So it looks better to me to change [`stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard/blob/b0430c00d34f44098f9c8668e1dad0f9c9a4b47a/index.js#L169) instead. What do you think?

**Comment by jeddy3** [comment]
> This issue seems related to not only [value-no-vendor-prefix](https://stylelint.io/user-guide/rules/list/value-no-vendor-prefix) but also [property-no-vendor-prefix](https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix) because the -webkit-line-clamp property has also been standardized for legacy compatibility.

If they're been standardized for legacy compatibility then we should probably fix the behaviour of the rules so that they're not flagged as problematic.

**Comment by ybiquitous** [comment]
I see. The next problem seems to be how to change the default behavior:

- change the default value of `ignoreValues` as proposed
- or, ignore `-webkit-box` regardless of `ignoreValues`

**Comment by jeddy3** [comment]
> change the default value of ignoreValues as proposed

Let's do this one.

**Comment by ybiquitous** [comment]
If people want to ignore more values in addition to the default one, they will need to include it like this:

```json5
{
  "ignoreValues": [
    "box", // default
    "other",
    "values"
  ]
}
```

this way may take some doing, but indeed, it is more flexible than the latter way; "ignore `-webkit-box` regardless of `ignoreValues`".

I agree with the way "change the default value of `ignoreValues`". 👍🏼

**Comment by jeddy3** [comment]
> So it looks better to me to change [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard/blob/b0430c00d34f44098f9c8668e1dad0f9c9a4b47a/index.js#L169) instead

Sorry for the confusion. Let's do it this way, i.e. make the change in the standard config.
